nominees/covid-19-api.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Covid-19 API",
+  "description": "This API can be utilised by all applications that need to write/retrieve Ethiopia Covid19 data. E.g. covid19.et project will use it to retrieve data for dashboard and write data",
+  "website": "https://github.com/africa-covid-19-response-toolkit/api",
```

Comment:
@rachita19082 kindly update the website field to this [link](https://ethiocovid19rt.com/)